roles/splunk_common/tasks/enable_forwarder_monitoring.yml

```diff
@@ -0,0 +1,22 @@
+- name: Enable forwarder monitoring
+  ini_file:
+    path: "{{ splunk.home }}/etc/apps/splunk_monitoring_console/local/savedsearches.conf"
+    section: "DMC Forwarder - Build Asset Table"
+    option: "{{ item.key }}"
+    value: "{{ item.value }}"
+  with_dict: {"cron_schedule": "{{ dmc_asset_interval }}", "alert.track": 0, "disabled": 0}
+  notify:
```

Comment:
Is this something we can also make a REST call to avoid the restart?

Reply:
I tried out the REST call but was unable to even populate the local/savedsearches.conf, even with a restart.